C/main_libFuzzer_extra_counters.c

```diff
@@ -0,0 +1,38 @@
+#include <stdio.h>
+#include <stddef.h>
+#include <stdlib.h>
+#include <stdint.h>
+#include <string.h>
```

Comment:
How is this file involved in the build? I fail to understand how it becomes part of the build.